src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,37 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int TWELVE = 12;
+    public static final int MINUS_ONE = -1;
```

Comment:
Stała wykorzystywana jest w dwóch metodach, mimo że nie jest to ta sama stała. Powinny być to dwie osobne.